docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-pytorch.rst

```diff
@@ -0,0 +1,114 @@
+Quantization-aware Training (QAT) with PyTorch
+===============================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
+PyTorch:
+
+1. Import NNCF API
+########################
+
+In this step, you add NNCF-related imports in the beginning of the training script:
+
+.. tab:: PyTorch
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [imports]
+
+
+2. Create the data transformation function
+################################################
+
+In the next step, you need to create a quantization data loader and wrap it by the nncf.Dataset, specifying a transformation
+function which prepares input data to fit into model during quantization.
+
+.. tab:: PyTorch
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [nncf_dataset]
+
+
+3. Apply optimization methods
+####################################
+
+nncf.quantize function accepts model and prepared quantization dataset for performing basic quantization. Optionally,
+additional parameters like subset_size, preset, ignored_scope can be provided to improve quantization result if applicable.
+More details about supported parameters can be found on this `page <https://docs.openvino.ai/2024/openvino-workflow/model-optimization-guide/quantizing-models-post-training/basic-quantization-flow.html#tune-quantization-parameters>`__.
+
+.. tab:: PyTorch
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [quantize]
+
+
+4. Fine-tune the model
+########################
+
+This step assumes that you will apply fine-tuning to the model the same way as it is done for the baseline model. In the
+case of QAT, it is required to train the model for a few epochs with a small learning rate, for example, 10e-5. In principle,
+you can skip this step which means that the post-training optimization will be applied to the model.
+
+.. tab:: PyTorch
+
+   .. doxygensnippet:: docs/optimization_guide/nncf/code/qat_torch.py
+      :language: python
+      :fragment: [tune_model]
+
+
+
+5. Export quantized model
+####################################
+
+When fine-tuning finishes, the quantized model can be exported to the ONNX format for further inference.
```

Comment:
Do we support the direct convert to OV IR via PTFE and OVC?

Reply:
Yes, we support this, but a separate section with a link to a separate file is describing the conversion methods in the [current version]( http://openvino-doc.iotg.sclab.intel.com/nncf-docs/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-pytorch.html#deploying-quantized-model) of the document. Should we remove this section from the doc?

Reply:
No, I think we should explain the recommended path for it. @AlexKoff88 , what do you think? Should we switch to OVC here?

Reply:
I recommend referring to the relevant OpenVINO product documentation in that aspect.

Reply:
> No, I think we should explain the recommended path for it. @AlexKoff88 , what do you think? Should we switch to OVC here?
I think we can use the code snippet from http://openvino-doc.iotg.sclab.intel.com/nncf-docs/openvino-workflow/model-optimization-guide/quantizing-models-post-training/basic-quantization-flow.html

Reply:
Should I use a code snipped with conversation to OV together with the link to the OV conversation documentation?

Reply:
I removed the export section and updated the deployment section with a small code snipped